### DISCUSSION TITLE
Sensitivity calculation with respect to SST model parameters

### DISCUSSION THREAD

**Discussion by itopcuoglu** [su2code/SU2#1804] [Q&A]
Hi,

I would like to know if it is possible to calculate sensitivities in SU2 with respect to turbulence model closure parameters. I would like to do this for the SST model. If generic parameter definition for sensitivities is not enabled, I would appreciate any tips on implementing this.

Thank you

**Comment by pcarruscag** [comment]
Hi, have a look at https://github.com/su2code/SU2/issues/679
